### PR TITLE
Update links to Get into teaching

### PR DESCRIPTION
**Pull request by frankieroberto**
@EllieNodder @avinhurry and @EmmaFrith  reviewed links to the Get into teaching service, discovering some that have changed and others that have moved. This updates the links and improves the link text.

Additionally, all the URLs have been moved to the `en.yml` locale file, to make it easier for us to audit where we are linking to in future.

[Trello card](https://trello.com/c/Ihs4Ib6P/4318-edit-links-from-find-and-apply-to-git)

### CODE REVIEW

**Review by EmmaFrith — APPROVED** [review]
thanks Frankie!

File: app/views/courses/financial_support/_scholarship_and_bursary.html.erb [hunk 1]
```diff
@@ -24,15 +24,15 @@
...
-      You may also be eligible for a <%= govuk_link_to 'loan while you study', 'https://getintoteaching.education.gov.uk/funding-your-training#tuition-fee-and-maintenance-loans' %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
+      You may also be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
     </p>
   <% else %>
     <p class="govuk-body">
-      You may be eligible for a <%= govuk_link_to 'loan while you study', 'https://getintoteaching.education.gov.uk/funding-your-training#tuition-fee-and-maintenance-loans' %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
+      You may be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
```
Comment (EmmaFrith) [inline app/views/courses/financial_support/_scholarship_and_bursary.html.erb:35]:
```suggestion
      You may be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %>.
```

File: app/views/courses/financial_support/_scholarship_and_bursary.html.erb [hunk 2]
```diff
@@ -24,15 +24,15 @@
...
+      <%= govuk_link_to 'Find out more about bursaries and scholarships', t('get_into_teaching.url_bursaries_and_scholarships') %>. You do not need to apply for a bursary - if you’re eligible, you’ll automatically start receiving it once you begin your course.
     </p>
 
     <p class="govuk-body">
-      You may also be eligible for a <%= govuk_link_to 'loan while you study', 'https://getintoteaching.education.gov.uk/funding-your-training#tuition-fee-and-maintenance-loans' %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
+      You may also be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
```
Comment (EmmaFrith) [inline app/views/courses/financial_support/_scholarship_and_bursary.html.erb:31]:
```suggestion
      You may also be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %>.
```

File: app/views/courses/financial_support/_loan.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div data-qa="course__loan_details">
   <p class="govuk-body">
-    You may be eligible for a <%= govuk_link_to 'loan while you study', 'https://getintoteaching.education.gov.uk/funding-your-training#tuition-fee-and-maintenance-loans' %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
+    You may be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
```
Comment (EmmaFrith) [inline app/views/courses/financial_support/_loan.html.erb:3]:
```suggestion
    You may be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %>.
```

**Review by avinhurry — APPROVED** [review]

File: app/components/courses/financial_support/bursary_component.html.erb [hunk 4]
```diff
@@ -14,13 +14,11 @@
...
     <p class="govuk-body">
       You do not have to apply for a bursary - if you’re eligible, you’ll automatically start receiving it once you begin your course.
-      Find out about <%= govuk_link_to 'eligibility', 'https://getintoteaching.education.gov.uk/funding-your-training#bursaries-and-scholarships' %> and
-      <%= govuk_link_to 'how you’ll be paid', 'https://getintoteaching.education.gov.uk/guidance/become-a-teacher-in-england#how-to-get-a-bursary' %>.
     </p>
   <% end %>
 
   <p class="govuk-body">
-    You may be eligible for a <%= govuk_link_to 'loan while you study', 'https://getintoteaching.education.gov.uk/funding-your-training#tuition-fee-and-maintenance-loans' %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
+    You may be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
```
Comment (EmmaFrith) [inline app/components/courses/financial_support/bursary_component.html.erb:21]:
```suggestion
    You may be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %>.
```

File: app/components/courses/financial_support/scholarship_and_bursary_component.html.erb [hunk 5]
```diff
@@ -24,15 +24,15 @@
...
-      You may also be eligible for a <%= govuk_link_to 'loan while you study', 'https://getintoteaching.education.gov.uk/funding-your-training#tuition-fee-and-maintenance-loans' %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
+      You may also be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
     </p>
   <% else %>
     <p class="govuk-body">
-      You may be eligible for a <%= govuk_link_to 'loan while you study', 'https://getintoteaching.education.gov.uk/funding-your-training#tuition-fee-and-maintenance-loans' %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
+      You may be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
```
Comment (EmmaFrith) [inline app/components/courses/financial_support/scholarship_and_bursary_component.html.erb:35]:
```suggestion
      You may be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %>.
```

File: app/views/courses/financial_support/_bursary.html.erb [hunk 6]
```diff
@@ -14,13 +14,11 @@
...
     <p class="govuk-body">
       You do not have to apply for a bursary - if you’re eligible, you’ll automatically start receiving it once you begin your course.
-      Find out about <%= govuk_link_to 'eligibility', 'https://getintoteaching.education.gov.uk/funding-your-training#bursaries-and-scholarships' %> and
-      <%= govuk_link_to 'how you’ll be paid', 'https://getintoteaching.education.gov.uk/guidance/become-a-teacher-in-england#how-to-get-a-bursary' %>.
     </p>
   <% end %>
 
   <p class="govuk-body">
-    You may be eligible for a <%= govuk_link_to 'loan while you study', 'https://getintoteaching.education.gov.uk/funding-your-training#tuition-fee-and-maintenance-loans' %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
+    You may be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %> – note that you’ll have to apply for <%= govuk_link_to 'undergraduate student finance', 'https://www.gov.uk/student-finance' %>.
```
Comment (EmmaFrith) [inline app/views/courses/financial_support/_bursary.html.erb:21]:
```suggestion
    You may be eligible for a <%= govuk_link_to 'loan while you study', t('get_into_teaching.url_funding_your_training') %>.
```

**Pull request merged**